python/ql/test/experimental/dataflow/coverage/test.py

```diff
@@ -77,16 +75,14 @@ def test_imagnumber_literal():
     x = 42j
     SINK(x)  # Flow missing
 
-# 6.2.3. Parenthesized forms
-
 
+# 6.2.3. Parenthesized forms
 def test_parenthesized_form():
-    x = (SOURCE)
+    x = SOURCE
     SINK(x)
```

Comment:
with autoformatting, this test suddently became trivial :sweat_smile:  :disappointed:

Reply:
There are two new tests in there: https://github.com/github/codeql/commit/8f13d586b7662f39f60e9613aebeec511ebb861e#diff-601e823041b1218704b390c8387e5bb6R412-R426
I agree that they are near impossible to spot in that view though. Doing the autoformatting on its own is the right way, thanks for that suggestion. Would you like me to rebase to do that?

Reply:
Yes, autoformatting is dangerous like that... 😥